src/components/articles/viewArticles.jsx

```diff
@@ -20,75 +22,90 @@ class AllArticles extends Component {
   };
   render() {
     const { fetched } = this.props.state.articles;
-    const pageSize = 20
+    const pageSize = 20;
     const { currentPage } = this.state;
     let paginateArticles;
     switch (fetched) {
       case '':
         return (
-          <div id='component-ViewArticles'>
+          <div id="component-ViewArticles">
             <NavBar />
-            <div className='masonry-container'>
-              <div className='masonry'>
+            <div className="masonry-container">
```

Comment:
We agreed that we will use single quotes. Can you change back this to single quotes?